README.md

```diff
@@ -64,14 +66,17 @@ and may also be used independently outside Rails.
    Run with `--help` or `-h` for options.
 
 4. Using a browser, go to `http://localhost:3000` and you'll see:
-"Yay! You’re on Rails!"
 
-5. Follow the guidelines to start developing your application. You may find
-   the following resources handy:
-    * [Getting Started with Rails](http://guides.rubyonrails.org/getting_started.html)
-    * [Ruby on Rails Guides](http://guides.rubyonrails.org)
-    * [The API Documentation](http://api.rubyonrails.org)
-    * [Ruby on Rails Tutorial](http://www.railstutorial.org/book)
+![Welcome to Rails Page](guides/assets/images/welcome_page.png "Welcome to Rails")
```

Comment:
This is out of date for master branch - the rails 5 welcome page has been changed